dbc2val/Dockerfile

```diff
@@ -31,35 +33,26 @@ ENV PATH="/opt/venv/bin:$PATH"
 RUN /opt/venv/bin/python3 -m pip install --upgrade pip \
     && pip3 install --no-cache-dir -r requirements.txt
 
-RUN pip3 install wheel scons && pip3 install pyinstaller patchelf==0.17.0.0 staticx
+RUN pip3 install wheel scons && pip3 install pyinstaller 
```

Comment:
Based on our recent problems with pyinstaller (well actually with staticx it seems) I am thinking if we in the long run should handle the dependencies here similar to our requirements* files, i.e. directly or indirectly specify exact version so we can always "rebuild" the same docker container. Maybe not for main/master, but maybe for releases.

Reply:
Should we add it to dev-dependencies? Or create a seperate "requirements-package.in" or something, generate it and pip-install it as well?